src/tests/ftest/server/cpu_usage.yaml

```diff
@@ -1,7 +1,24 @@
 hosts:
   test_servers:
     - server-A
-timeout: 80
+  test_clients:
+    - client-A
+timeout: 130
 server_config:
```

Comment:
Just a thought.. 
So now we are changing this test maybe we should have all the base components like NVMe/DCPMEM also involve in the test so we can have a more robust baseline with memory even it doe not impact the CPU as much.

Reply:
Okay. I'll add more variants to test the combination of socket, verbs, NVMe, and PMEM.

Reply:
My intent was not to ask for adding more variants but to have a single test variant and we can use all the resources we have in Ci. So for example to move this test on the small cluster and use whatever provider we have (Sockets for now)  and add DCPMEM and NVMe.
So If we have that test not sure if we wanted to run the same test on VM, make sense?

Reply:
Okay. I'll let it use all the resources in a single test case. Not sure why you say verbs later. I think it's better to test with verbs since that's what users would use.

Reply:
I mean we do not have verbs HW environment in Ci yet so when HW is moved to Mellanox IB that is when we will use the full IB stack.